draft-ietf-rtcweb-jsep.xml

```diff
@@ -266,13 +266,11 @@ ipr="trust200902">
         what the local side is willing to receive, which, when
         intersected with the set of codecs the remote side supports,
         specifies what the remote side should send. However, not all
-        parameters follow this rule; for example, the DTLS-SRTP
-        parameters
-        <xref target="RFC5763"></xref> sent to a remote party indicate
-        what certificate the local side will use in DTLS setup, and
-        thereby what the remote party should expect to receive; the
-        remote party will have to accept these parameters, with no
-        option to choose different values.</t>
+        parameters follow this rule; for example, the fingerprints
+        <xref target="I-D.ietf-mmusic-4572-update"></xref>
+        sent to a remote party are calculated based on the local
+        certificate(s) offered; the remote party will have to accept
+        these parameters, with no option to choose different values.</t>
```

Comment:
PR updated. I used "reject the parameter" instead of "reject the offer", because one can reject one m- line without rejecting the whole offer. Or, does JSEP require accept-all-or-nothing?